c++/Nbody.cpp

```diff
@@ -0,0 +1,166 @@
+#pragma once
+#include <iostream>
+#include <cmath>
+#include <vector>
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <cmath>
+#include <chrono>
+
+
+using namespace std;
+
+ const  double G = 0.6;//6.67428e-11;
+ const  double AU = 149.6e6 * 1000;
```

Comment:
Double spaces here

Reply:
Where is PEP8 detection when you need it?

Reply:
I am pep8, and I have detected